server/chat-plugins/responder.ts

```diff
@@ -441,7 +441,11 @@ export const pages: PageTable = {
 		let buf = '';
 		const refresh = (type: string, extra?: string[]) => {
 			let button = `<button class="button" name="send" value="/join view-autoresponder-${room.roomid}-${type}`;
-			button += `${extra ? `-${extra.join('-')}` : ''}" style="float: right">`;
+			if (extra) {
+				button += `${extra.length ? `-${extra.join('-')}` : ''}" style="float: right">`;
+			} else {
+				button += `" style="float: right">`;
+			}
```

Comment:
```suggestion
			if (extra?.length) button += `-${extra.join('-')}`;
			button += `" style="float: right">`;
```
I agree with Mia's suggestion on the need for `.filter(Boolean)`; an empty array or `[""]` would still cause the extra hyphen to be appended.

Reply:
(See my most recent code block comment, should cover this)

Reply:
Although it looks like your code on line 462 prevents those arrays from being passed in, unless `refresh()` is called elsewhere.